src/signals/incident-management/containers/StandardTextsAdmin/components/DetailPage/Detail.test.tsx

```diff
@@ -0,0 +1,183 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2023 Gemeente Amsterdam
+import { render, screen, waitFor } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import * as reactRedux from 'react-redux'
+
+import { showGlobalNotification } from 'containers/App/actions'
+import { withAppContext } from 'test/utils'
+
+import { Detail } from './Detail'
+import type { StandardTextDetailData } from './types'
+import * as API from '../../../../../../../internals/testing/api'
+import {
+  fetchMock,
+  mockRequestHandler,
+} from '../../../../../../../internals/testing/msw-server'
+import {
+  TYPE_LOCAL,
+  VARIANT_ERROR,
+} from '../../../../../../containers/Notification/constants'
+import { mockSubcategory } from '../SelectedSubcategories/mock-subcategories'
+
+fetchMock.disableMocks()
+const mockNavigate = jest.fn()
+const dispatch = jest.fn()
+
+const id = 4
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useParams: () => ({ id }),
+  useNavigate: () => mockNavigate,
+}))
+
+const mockData: StandardTextDetailData = {
+  id: 4,
+  active: true,
+  categories: [79, 4],
+  created_at: '24-06-2023',
+  state: 'o',
+  text: 'Mooie omschrijving',
+  title: 'Mooie titel',
+  updated_at: '25-06-2023',
+}
+
+describe('Detail', () => {
+  beforeEach(() => {
```

Comment:
Use adequate spacing for readability.

Reply:
Added space just about everywhere, readability is a very nice to have